style.css

```diff
@@ -0,0 +1,103 @@
+@media only screen and (max-width: 765px) {
+  body {
+    background-color: black;
+    max-height: 1000px;
+    margin: 0;
+  }
+
+  header {
+    background-color: black;
+  }
+
+  .home {
+    background-image: url("images/Group.png");
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: bottom right;
+    display: flex;
+  }
```

Comment:
- [ ] Good job applying `flex` as requested by the previous reviewer. When doing so, I think you forgot to add `flex-direction: column;` and the design is looking like this at the moment:
<img src="https://user-images.githubusercontent.com/17243325/204971333-3ecaa373-36ea-476a-8030-8d1dde0311ef.png" width="300">

Kindly add the `flex-direction` so your elements can stack from top to bottom instead of left to right. 🤝️